README.md

```diff
@@ -437,6 +437,7 @@ nodeunit functionality is available.
 
 To run the nodeunit tests do:
 
+    npm install ejs should
```

Comment:
`npm i` (short for `npm install`) would be normal procedure, right?

Reply:
Ah yes, you're right. I was new to node when I wrote that. I've resubmitted.